bundles/ch.elexis.core.ui/src/ch/elexis/core/ui/views/TextView.java

```diff
@@ -225,6 +228,40 @@ public boolean createDocument(Brief template, String subject, Kontakt adressat)
 			return false;
 		}
 		actBrief = txt.createFromTemplate(Konsultation.getAktuelleKons(), template, Brief.UNKNOWN, adressat, subject);
+
+		String adressatLabel = null;
```

Comment:
Bitte diesen Vorgang in eine eigene Methode auslagern, und diese an dieser Stelle aufrufen!